app/models/payment.rb

```diff
@@ -0,0 +1,5 @@
+# coding: utf-8
+# frozen_string_literal: true
+class Payment < ActiveRecord::Base
```

Comment:
Could be useful to document the requirements & usage of `amount` here. I think it's necessary to make it very discoverable that the value must be a whole number of halalas ("cents") rather than a fractional representation of riyals ("dollars")